tools/flang2/flang2exe/cgmain.cpp

```diff
@@ -1674,6 +1688,11 @@ schedule(void)
     } else {
       clear_rw_access_grp();
     }
+    if (flg.x[234] > 0) {
+      BIH_VECTORLENGTH_ENABLED(bih) = true;
+      BIH_VECTORLENGTH_SCALABLE( bih ) = (XBIT(234, 0x4)>0);
```

Comment:
I don't think you need the `>0` here. I think `XBIT` will return you a nice 1-bit value to store here.